### PR TITLE
[DOCS] Add info about mailer in installation doc

### PR DESCRIPTION
**Pull request by arti0090**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         |  1.12           |
| License         | MIT                                                          |

### CODE REVIEW

File: docs/book/installation/installation.rst [hunk 1]
```diff
@@ -61,6 +61,24 @@ After everything is in place, run the following command to install Sylius:
     to change these before running the installation command, set the ``locale`` and ``sylius_installer_currency`` parameters in the ``config/services.yaml`` file.
     From now on all the prices will be stored in the database in USD as integers, and all the products will have to be added with a base american english name translation.
 
+Configuring Mailer
+------------------
+
+In order to send emails you need to configure Mailer Service. Basically you need to:
+
+1. **Create an account on a mailing service.**
+2. **In your** `.env` file **modify variable** ``MAILER_URL``
```
Comment (CoderMaggie) [inline docs/book/installation/installation.rst]:
```suggestion
2. **In your** ``.env`` **file modify the** ``MAILER_URL`` **variable.**
```

**Review by CoderMaggie — APPROVED** [review]

**Review by lchrusciel — APPROVED** [review]

**Review by GSadee — COMMENTED** [review]
This PR seems to be a little bit outdated, probably it should be merged to the 1.11 branch in this version and be updated in 1.12 because of [the PR](https://github.com/Sylius/Sylius/pull/14341)

**Comment by GSadee** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "mailer-install"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Review by lchrusciel — APPROVED** [review]
but we should already update this doc for 1.12 and Symfony Mailer

**Pull request merged**